web/book/src/tutorial/aggregation.md

```diff
@@ -1,95 +1,98 @@
 
 ```
-from albums
-aggregate {sum_of_ids = sum album_id}
+from invoices
+aggregate { sum_of_orders = sum total }
 ```
+In the example above, the `total` column of all rows of the `invoices` table
+is summed to produce a single value.
 
-`aggregate` is quite similar to `derive` and `select` as it is creating a new
-column, with the option to assign it a name. Just as select, it will discard all
-columns that are not present in the tuple, but with a key difference: it will
-always produce a single row.
-
-Just as `select` (and actually `derive` too), `aggregate` takes a tuple of
-aggregation expressions, which means that you can produce multiple summaries at
-once:
+`aggregate` can produce multiple summaries at once when
+one or more aggregation expressions are contained in a tuple.
+`aggregate` discards all columns that are not present in the tuple.
 
 ```
-from albums
+from invoices
 aggregate {
-    sum_of_ids = sum album_id,
-    mean_id = average album_id,
+    num_orders = count this,
+    sum_of_orders = sum total,
 }
 ```
 
-Here we've also split the tuple over multiple lines to make it more readable.
-Obviously, you can do this with `select` and `derive` too.
+In the example above, the result is a single row with two columns.
+The `count` function displays the number of rows in the table that was passed in;
+the `sum` function adds up the values of the `total` column of all rows.
 
 ## Grouping
 
-Say that now we want to produce summaries of albums _for each artist
-separately_. We cannot do this, as `aggregate` will always produce a single row.
+Suppose we want to produce summaries of invoices _for each city_
+in the table.
+`aggregate` cannot do this because it will always produce a single row.
 
-What we can do, is separate the relation into groups corresponding to individual
-artists, and apply `aggregate` to each one of them:
+We could separate the relation into groups corresponding to individual
+cities, and apply `aggregate` to each one of them:
 
 ```
 from albums
-filter artist_id == 1
-aggregate {sum_of_ids = sum album_id}
+filter billing_city == "Oslo"
+aggregate { sum_of_orders = sum total }
 ```
 
 ```
 from albums
-filter artist_id == 2
-aggregate {sum_of_ids = sum album_id}
+filter billing_city == "Frankfurt"
+aggregate { sum_of_orders = sum total }
 ```
 
 ```
 from albums
-filter artist_id == 3
-aggregate {sum_of_ids = sum album_id}
+filter billing_city == "London"
+aggregate { sum_of_orders = sum total }
 ```
 
-... and so on. Of course this is not the way to go, because it is way too
-repetitive and we need to type all of the artist ids by hand. Luckily, `group`
-will do exactly what we want: separate relation into groups, apply a function to
-each of the groups and combine the results back together:
+... and so on. Of course this is repetitive (and boring) and error prone (because we would need to type each `billing_city` by hand). Moreover, we would need to create the `billing_city` list before we started.
```

Comment:
Not a verdict — feel free to ignore — but I think this is very laborious given the average reader — they probably know about group-by from SQL, or at least the concept of a pivot table in Excel.

So I would skip this explanation and just jump into the concept of grouping...

Reply:
For newcomers, I think it's important to motivate the notion of grouping. (The experts will only be reading the Tutorial to see if they're smarter than we are :-) 

That said, I can trim it down to one example PRQL query and then describe London and Oslo, etc.